.github/workflows/benchmarks.yml

```diff
@@ -93,7 +93,7 @@ jobs:
 
       - name: Store benchmark result
         uses: benchmark-action/github-action-benchmark@v1
-        if: github.ref == 'refs/heads/master'
+        # if: github.ref == 'refs/heads/master'
```

Comment:
I think that here we should keep the `if`. Only store result from pushes to master